code/testsuite/csheets/pf_Cleric.xml

```diff
@@ -1566,9 +1540,35 @@ BIO
 			<weight>4</weight>
 			<bonuslist></bonuslist>
 		</item>
+		<item>
+			<longname>Holy Symbol, Silver</longname>
+			<id></id>
+			<name>Silver Holy Symbol</name>
+			<carried>1.0</carried>
+			<charges></charges>
+			<charges_used></charges_used>
+			<contents></contents>
+			<contents_num>0</contents_num>
+			<content_weight>0</content_weight>
+			<cost>25</cost>
+			<equipped>Y</equipped>
+			<location>Equipped</location>
+			<maxcharges>0</maxcharges>
+			<note></note>
+			<quantity>1</quantity>
+			<checkbox>1</checkbox>
+			<size>
+				<long>Medium</long>
+				<short>M</short>
+			</size>
+			<special_properties></special_properties>
+			<type>GOODS.TOOLS.SPELL COMPONENT.DIVINE FOCUS.HOLY SYMBOL</type>
+			<weight>1</weight>
+			<bonuslist></bonuslist>
+		</item>
 		<total>
 			<weight>59.5 lbs.</weight>
-			<value>926.6 gp</value>
+			<value>901.6 gp</value>
```

Comment:
Why did this value change on a test character?

Reply:
The amount off is the value of a silver holy symbol (25 gp). I was attempting to reduce the holy symbol to a single mod-required item, but the migration file doesn't allow EQMODs to be added. I thought I had fixed it, but apparently not.